workos/src/main/kotlin/com/workos/common/http/PaginationParams.kt

```diff
@@ -20,17 +27,37 @@ open class PaginationParams @JvmOverloads constructor(
     }
   }
 
+  /**
+   * @suppress
+   */
   companion object {
     @JvmStatic
-    fun builder(): Builder<PaginationParams> {
-      return Builder(PaginationParams())
+    fun builder(): PaginationParamsBuilder<PaginationParams> {
+      return PaginationParamsBuilder(PaginationParams())
     }
   }
 
-  open class Builder<T : PaginationParams>(protected val params: T) {
+  /**
+   * Builder class for creating [PaginationParams]
```

Comment:
Missing a period, I think.
```suggestion
   * Builder class for creating [PaginationParams].
```